client/layout/index.js

```diff
@@ -66,6 +67,8 @@ class Layout extends Component {
 				<TransientNotices />
 
 				<div className="woocommerce-layout__primary" id="woocommerce-layout__primary">
+					<StoreAlerts />
```

Comment:
Prior to merging this in, I think we should add a feature flag around this so it only shows up in Dev mode. #1500 has some examples on how that is done.